changelog.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## NEXT
+
+- Fixed a bug in `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph.from_causal_graph` for
+  `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph` where the method was not adding floating nodes
+  correctly.
+- Added `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph.max_backward_lag` and
+  `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph.min_forward_lag` properties to
+  `cai_causal_graph.time_series_causal_graph.TimeSeriesCausalGraph` to return the absolute backward and forward time
```

Comment:
" ... return the absolute maximum backward and forward ..." - just add the work maximum